cmd/prometheus-config-reloader/main.go

```diff
@@ -76,6 +77,11 @@ func main() {
 		"address on which to expose metrics (disabled when empty)").
 		String()
 
+	webConfig := app.Flag(
+		"web-config-file",
+		"Path to configuration file that can enable TLS or authentication. See: https://github.com/prometheus/exporter-toolkit/blob/master/docs/web-configuration.md",
```

Comment:
(nit) might be better to link to https://prometheus.io/docs/prometheus/latest/configuration/https/

Reply:
Thanks you for your review. Done